### ISSUE TITLE
Outlook Add-in Centralized Deployment - Not working

### ISSUE THREAD

**Issue by ak-19** [OfficeDev/office-js#4564]
# Outlook Add-in Centralized Deployment - Not working!!
1. Using "Microsoft 365 admin center > Settings > Integrated Apps > Upload custom apps > Office addin > Provide link to manifest file"
Trying update manifest via URL - Not working
Deleting app and creating new one - Not working
deployment ends with:
![dep-fail](https://github.com/OfficeDev/office-js/assets/382654/2ebc5ddc-5602-4a87-9039-0a1e8667bab4)
2. Using "Adminportal/Home/#/Settings/AddIns" and got:
![ERROR-SCR](https://github.com/OfficeDev/office-js/assets/382654/f74826be-7236-40d0-b574-a8f6541cfef9)

Everything worked over 2 Years and it is not working at the moment, we have customers deploying outlook add-in via Centralized Deployment, now we are blocked with this issue.

## Environment
Platform [Windows desktop, Mac desktop, Office on the web]: Mac/Windows
Operating System: Windows/MacOS
Browser: All browsers
## Expected behavior
Update should work with the same manifest using "update manifest via URL" feature.
Create custom app should work with the manifest fiile using "Provide link to manifest file" feature.

If you need more details from our side, please tell me in a comments.

**Comment by ak-19** [comment]
I tried to run that tool via powershell "Invoke-CompatibilityCheck", that thing fails to login with admin creadentials.

**Comment by mobisw-msft** [comment]
Hey @ak-19 ,

To help you further we will need the tenant details of the tenant you're deploying the add-in to, and the manifest that you are trying to deploy. You can share it by creating a private github repository with the details and giving us access to the repository.

**Comment by ak-19** [comment]
Name: NETSTAG GmbH
Tenant ID: 2b6088bd-6ef1-47f7-9580-492519bd93ef
License: Microsoft Entra ID Free

**Comment by ak-19** [comment]
Manifest is within private repo, user "mobisw-msft" has pending invite to acces it

**Comment by mobisw-msft** [comment]
Hey @ak-19 , When validating your manifest, we are encountering a validation failure "Icon incorrectly Sized - Your icon Width*Height is: 32*32. Icon size should be 64*64". Can you please work on this lineitem and see if the add-in works?

**Comment by ak-19** [comment]
Yesterday I was able to create new subscription and sucessfully deploy this manifest on it, but on my company subscription this is not working as described above.
Also I tried to make new build with manifest that has 64x64 icon and same thing happens, not able to deploy on my company tenant but it works when I create trial subscription and deploy there.

**Comment by ajnavsun1806** [comment]
@ak-19 I checked the logs for ID 3ec53d9a-1ac1-46d9-9646-6b106f0a2c79 as shown in screenshot.
Error recorded in our system is below:
Box Exception Occurred: You do not have permission to install custom add-in for organization.

Please check if there are some changes in the Default Role Assignment policy in EAC which is blocking you from deploying custom add-in.

**Comment by ak-19** [comment]
Yes but this worked previously and issues appears on othere tenatns too (some of our customers).
Also user doing deployment is Global administrator.

**Comment by PatrickGIHUB** [comment]
I have the exact same issue in our tenat. Every other tenant the manifest.xml works fine.
Do you have any update on this problem?
